File: C/mycps/1stSem/palindrome.c
#include<stdio.h>

int main()
{
	int i, orgn, n, revn=0, newn;

	printf("Enter a Number:");

	scanf("%d",&i);

	orgn = i;

	while (orgn>0)
	{
		n = orgn % 10;

		revn = (revn*10) + n;

		orgn  = orgn / 10;
	}

	if (revn == i)
		printf("Entered No. is a Palindrome\n");

	else
		printf("Entered No. is not a Palindrome\n");
}

File: Java/Assignment 2/Palindrome.java
import java.util.Scanner;

class Palindrome
{
	public static int pal(int n)
	{
		int revn=0;
		while(n>0)
		{
			int digit = n % 10;
			revn = (revn * 10) + digit;
			n /= 10;
		}
		return revn;
	}
	
	public static void main(String[] args)
	{
		Scanner sc = new Scanner(System.in);
		
		System.out.print("Enter a number to check palindrome: ");
		int n = sc.nextInt();
		
		int revn = pal(n);
		
		if (n == revn)
			System.out.println("Entered number is a Palindrome.");
		else
			System.out.println("Entered number is not a Palindrome.");
	}
}